.github/workflows/check-if-migration-is-required.yml

```diff
@@ -0,0 +1,88 @@
+name: check if migration is required
+on:
+  pull_request:
+    paths:
+      # Matches all python files regardless of directory depth.
+      - '**.py'
+      - requirements*.txt
+
+jobs:
+  main:
+    runs-on: ubuntu-16.04
+
+    env:
+      PIP_DISABLE_PIP_VERSION_CHECK: on
+      SENTRY_LIGHT_BUILD: 1
+      SENTRY_SKIP_BACKEND_VALIDATION: 1
+      MIGRATIONS_TEST_MIGRATE: 0
+
+      # The hostname used to communicate with the PostgreSQL from sentry
+      DATABASE_URL: postgresql://postgres:postgres@localhost/sentry
+
+    services:
+      postgres:
+        image: postgres:9.6
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - name: Install System Dependencies
```

Comment:
I'm not planning on refactoring this to use `setup-sentry` in this PR

Reply:
What about path-filter?

Reply:
If you do do path-filter here, move the checkout to before the system dependency update.

Reply:
I'd prefer to do changes together w/ `setup-sentry` refactor